contracts/interfaces/ITreasuryExtender.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.8.10;
+
+struct AllocatorPerformance {
+    uint128 gain;
+    uint128 loss;
+}
+
+struct AllocatorLimits {
+    uint128 allocated;
+    uint128 loss;
+}
+
+struct AllocatorHoldings {
```

Comment:
Well, I believe inherently under the hood this is going to be mapping to a location in memory which is going to have the consecutive storage slots... associating it in a struct with a name doesn't change the underlying operation since it's just one storage slot with uint256, so this is done from a readability perspective similarly to splitting files up into folders...